package.json

```diff
@@ -22,52 +22,49 @@
     "lint:apply": "npm run format && npx eslint --fix '{src,test}/**/*.ts'",
     "format": "prettier --write '{,@(src|test)/**/}*.*'",
     "test:dev": "export NODE_ENV=test && TS_NODE_FILES=true ava --verbose --color --serial --watch",
-    "test": "npm run lint && npm run build && export NODE_ENV=test && nyc ava --verbose --color --serial --timeout=2m && nyc report --reporter=html",
+    "test": "npm run lint && npm run build && export NODE_ENV=test && nyc ava --verbose --color --serial --timeout=2m --no-worker-threads && nyc report --reporter=html",
     "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
     "preversion": "npm test",
     "prepublishOnly": "npm run build",
     "release": "export BABEL_DISABLE_CACHE=1 && release-it --ci",
     "release:dry": "export BABEL_DISABLE_CACHE=1 && release-it --ci --dry-run=true",
-    "post:release": "echo Successfully released $npm_package_version"
+    "post:release": "echo Successfully released $npm_package_version",
+    "prepare": "husky install"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/neo9/n9-node-log"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=16.0.0"
   },
   "dependencies": {
-    "pino": "^7.11.0",
-    "pino-pretty": "^7.6.1"
+    "pino": "^8.15.0",
```

Comment:
a new version of pino has been release (8.15.1), could you refresh upgrade to latest ?